### DISCUSSION TITLE
Achievenments

### DISCUSSION THREAD

**Discussion by JillValentine-br** [community/community#37127] [Other Feature Feedback, Questions, & Ideas]
Using the translator ...
Any video that teaches how to get a badge?

**Comment by FatChicken277** [comment]
This [repo](https://github.com/Schweinepriester/github-profile-achievements) is a collection listing all Achievements available on the GitHub profile and how to get it
The only video that i found about how to get the achievements [video](https://www.youtube.com/watch?v=wBW-vjM7ccA)

If I answered your question, then you should close the question by click accept answer button :heart:

**Comment by SilasRodrigues19** [comment]
@JillValentine-br In this repo there is a collection with all the achievements and the levels of each one, explaining what you need to get:

[GitHub-Achievements](https://github.com/drknzz/GitHub-Achievements)

**Comment by Khushboqof** [comment]
https://www.youtube.com/watch?v=6WHibkv6cc0
